Package.swift

```diff
@@ -27,14 +27,14 @@ let package = Package(
     .library(name: "Views", targets: ["Views"]),
     ],
   dependencies: [
-    .package(url: "https://github.com/pointfreeco/swift-prelude.git", .revision("6e426b0")),
-    .package(url: "https://github.com/pointfreeco/swift-snapshot-testing.git", .exact("1.2.0")),
+    .package(url: "https://github.com/ianpartridge/swift-backtrace.git", .exact("1.1.0")),
+    .package(url: "https://github.com/pointfreeco/Ccmark.git", .branch("master")),
     .package(url: "https://github.com/pointfreeco/swift-html.git", .exact("0.2.1")),
+    .package(url: "https://github.com/pointfreeco/swift-prelude.git", .branch("swift-5")),
+    .package(url: "https://github.com/pointfreeco/swift-snapshot-testing.git", from: "1.5.0"),
     .package(url: "https://github.com/pointfreeco/swift-tagged.git", .revision("926e8e0")),
-    .package(url: "https://github.com/pointfreeco/swift-web.git", .revision("2c3d440")),
-    .package(url: "https://github.com/pointfreeco/Ccmark.git", .branch("master")),
+    .package(url: "https://github.com/pointfreeco/swift-web.git", .revision("swift-5")),
```

Comment:
should we merge all the deps in before merging this?

Reply:
Yeah I think so.

Reply:
Done!